DQM/L1TMonitor/src/L1TStage2BMTF.cc

```diff
@@ -74,6 +75,28 @@ void L1TStage2BMTF::bookHistograms(DQMStore::IBooker &ibooker, const edm::Run& i
   bmtf_hwQual_bx   = ibooker.book2D("bmtf_hwQual_bx"  , "HW Quality vs BX"      ,  20,   -0.5,  19.5,  5, -2.5, 2.5);
   bmtf_hwQual_bx->setTitle("; HW Quality; BX");
 
+  if (kalman) {
+    ibooker.setCurrentFolder(monitorDir);
+    kbmtf_hwDXY = ibooker.book1D("kbmtf_hwDXY", "HW DXY", 4, 0, 4);
+    kbmtf_hwPt2 = ibooker.book1D("kbmtf_hwPt2", "HW p_{T}2", 512, -0.5, 511.5);
+    bmtf_hwEta->setTitle("kbmtf_hwEta");
```

Comment:
Instead of resetting the title of all histograms you could make the title variable when you book the bmtf_* MEs.
In line 40 do this:
```
std::string histoPrefix = "bmtf";
if (kalman) {
  histoPrefix = "kbmtf";
}
bmtf_hwEta = ibooker.book1D(histoPrefix+"_hwEta", "HW #eta", 461, -230.5, 230.5);
bmtf_hwLocalPhi = ibooker.book1D(histoPrefix+"_hwLocalPhi", "HW Local #phi", 201, -100.5, 100.5);
...
```

Reply:
Note also that the setTitle after each booking from lines 55 onwards erases the histogram title and puts the axis labels instead. The axis labels can be set directly when booking and not setTitle is needed.
```
bmtf_hwEta_hwLocalPhi = ibooker.book2D("bmtf_hwEta_hwLocalPhi", "HW #eta vs HW Local #phi;HW #eta;HW Local #phi" , 461, -230.5, 230.5, 201, -100.5, 100.5);
```